### ISSUE TITLE
ElastiCache has no snapshot retention

### ISSUE THREAD

**Issue by semyonfox** [semyonfox/oghma#67]
## Problem
ElastiCache serverless cache `oghmacache` has `SnapshotRetentionLimit: 0`. No backups exist. Cache loss = loss of all rate limiting state and cached settings.

## Fix
Enable daily snapshots with at least 1-day retention:
```bash
aws elasticache modify-serverless-cache --serverless-cache-name oghmacache --snapshot-retention-limit 1 --region eu-north-1
```

**Comment by semyonfox** [comment]
Fixed: ElastiCache snapshot retention set to 7 days via `aws elasticache modify-replication-group --snapshot-retention-limit 7 --snapshotting-cluster-id oghma-valkey-001`
